packages/transaction-controller/src/api/simulation-api.ts

```diff
@@ -175,6 +175,9 @@ export type SimulationResponseTokenFee = {
   /** Account address that token should be transferred to. */
   feeRecipient: Hex;
 
+  /** Whether the gas fee is sponsored meaning the transfer is not required. */
+  isSponsored?: boolean;
```

Comment:
Can this be removed, I assume we don't have the same property in the response twice?